### PR TITLE
crud-1 display coordinates

### PR DESCRIPTION
**Pull request by Dabble362**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -92,7 +93,14 @@ class Game extends React.Component {
       const desc = move ? "Go to move #" + move : "Go to game start";
       return (
         <li key={move}>
-          <button onClick={() => this.jumpTo(move)}>{desc}</button>
+          <button onClick={() => this.jumpTo(move)}>{desc}</button>{" "}
```
Comment (jduffey) [inline src/index.js:96]: This is a test comment. Please resolve it. Thanks!

File: src/index.js [hunk 2]
```diff
@@ -69,6 +69,7 @@ class Game extends React.Component {
       history: history.concat([
         {
           squares: squares,
+          location: i,
```
Comment (jduffey) [inline src/index.js:72]: This is a test review. Please address. Thanks!

File: src/index.js (hunk 1 above)
Comment (Dabble362) [reply src/index.js:96]: Test comment observed! Thank YOU

**Review by jduffey — CHANGES_REQUESTED** [review]
Testing a review submission that has "Request changes" selected.

File: src/index.js (hunk 2 above)
Comment (Dabble362) [reply src/index.js:72]: This is great.  I appreciate testing! Thanks.

Comment (jduffey) [reply src/index.js:72]: Ok looks good, thanks for taking care of this.

**Review by jduffey — APPROVED** [review]

**Comment by Dabble362** [comment]
Thanks @jduffey for the help!

**Pull request merged**